lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -507,6 +517,70 @@ func (h *StorageHandlersImpl) VolumeJoin(params storage.VolumeJoinParams) middle
 	return storage.NewVolumeJoinOK().WithPayload(actualHandle.String())
 }
 
+// ExportArchive creates a tar archive and returns to caller
+func (h *StorageHandlersImpl) ExportArchive(params storage.ExportArchiveParams) middleware.Responder {
+	defer trace.End(trace.Begin(""))
+
+	var filterSpec vicarchive.FilterSpec
```

Comment:
Can we put this logic in a helper method? I believe there's a filter spec decoder in the archive package on upstream master. @matthewavery